### PR TITLE
[stable24] Show estimated export size and display exportability warnings

### PR DESCRIPTION
**Pull request by Pytal**
Backport of https://github.com/nextcloud/user_migration/pull/136

### CODE REVIEW

File: src/components/ExportSection.vue [hunk 1]
```diff
@@ -81,6 +81,7 @@
 					</template>
 					{{ t('user_migration', 'Export') }}
 				</Button>
+				<span v-if="estimatedSizeWithUnits" class="settings-hint">{{ t('user_migration', 'Estimated size: {estimatedSizeWithUnits}', { estimatedSizeWithUnits: this.estimatedSizeWithUnits }) }}</span>
```
Comment (come-nc) [inline src/components/ExportSection.vue:84]: Github workflows are complaining about the usage of `this` here, i do not know enough about vue to understand what’s going on but it seems this is not used in the other blocks around, so maybe this is indeed a mistake?

Comment (Pytal) [reply src/components/ExportSection.vue:84]: The lint warning will be fixed on main and backported https://github.com/nextcloud/user_migration/pull/182#issuecomment-1168982305

**Comment by Pytal** [comment]
> /rebase

Not needed apparently

**Pull request merged**